services/api/src/routes/users.js

```diff
@@ -88,26 +99,82 @@ router
     };
   })
   .use(requirePermissions({ endpoint: 'users', permission: 'write', scope: 'global' }))
+  .post('/:userId/re-invite', async (ctx) => {
```

Comment:
Wondering if maybe just one route can handle both tasks? and/or maybe we want to send out multiple invites at a time... just so that it works the same as /invite

Reply:
i looked at this and i think it getting a bit too complex if i merge it together and i dont want to increase the scope here in this PR. the main thing is that that we no longer have 2 models (user + invites), just one model (user).